packages/protocol/test/governance/network/governance.ts

```diff
@@ -2472,4 +2481,99 @@ contract('Governance', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#getProposalStage()', () => {
+    const proposalId = 1
+    const expectStage = async (expected: Stage, _proposalId?: number) => {
```

Comment:
Totally optional of course, but not a super big fan of this optional parameter handling